lib/restbase.js

```diff
@@ -307,6 +318,17 @@ RESTBase.prototype._request = function (req) {
             self.metrics.endTiming([statName + statusClass, statName + 'ALL'], startTime);
             throw err;
         });
+        if (match.permissions
```

Comment:
@Pchelolo, having information about what's checked further up could enable some optimizations. It also gives us a bit of information that lets us make sure that we don't send cookies to internal services if we persist the response and expose it without further access checks.